src/Oro/Bundle/PimDataGridBundle/Validator/Constraints/UniqueDatagridViewEntity.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Oro\Bundle\PimDataGridBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @author Pierre Jolly <pierre.jolly@akeneo.com>
+ * @copyright 2021 Akeneo SAS (http://www.akeneo.com)
+ * @license http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class UniqueDatagridViewEntity extends Constraint
+{
+    public $message = 'The same label is already set on another view';
```

Comment:
```suggestion
    public $message = 'pim_datagrid.column_configurator.label.unique_message';
```

And remove the message in *src/Oro/Bundle/PimDataGridBundle/Resources/config/validation/datagrid_view.yml*?